src/ui/localization/default_locale.js

```diff
@@ -0,0 +1,8 @@
+// @flow
+
+const defaultLocale = {
+    'FullscreenControl.EnterTooltip': 'Enter fullscreen',
+    'FullscreenControl.ExitTooltip': 'Exit fullscreen'
```

Comment:
Also wondering whether we need long namespaced ids if there are not that many strings to define, but it will be clearer when we define them for all controls.